protocol-gateway/azure-mqtt-protocol-gateway/pom.xml

```diff
@@ -0,0 +1,270 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2020 Contributors to the Eclipse Foundation
+   
+    See the NOTICE file(s) distributed with this work for additional
+    information regarding copyright ownership.
+   
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License 2.0 which is available at
+    http://www.eclipse.org/legal/epl-2.0
+   
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.eclipse.hono</groupId>
+    <artifactId>azure-protocol-gateway-example</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+
+    <name>Hono Azure IoT Protocol Gateway Example</name>
+    <description>A simple protocol gateway for connecting Azure IoT Hub compliant devices via MQTT to Eclipse Hono.
+    </description>
+    <url>https://www.eclipse.org/hono</url>
+    <inceptionYear>2020</inceptionYear>
+
+    <organization>
+        <name>Eclipse Foundation</name>
+        <url>https://www.eclipse.org/</url>
+    </organization>
+
+    <licenses>
+        <license>
+            <name>Eclipse Public License - Version 2.0</name>
+            <url>http://www.eclipse.org/legal/epl-2.0</url>
+            <comments>SPDX-License-Identifier: EPL-2.0</comments>
+        </license>
+    </licenses>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+
+        <classifier.spring.boot.artifact>exec</classifier.spring.boot.artifact>
+
+        <!-- configure timestamp format to be injected into legal docs -->
+        <maven.build.timestamp.format>yyyy-MM-dd</maven.build.timestamp.format>
+        <timestamp>${maven.build.timestamp}</timestamp>
+
+        <assertj-core.version>3.15.0</assertj-core.version>
+        <hono.version>1.3.0-SNAPSHOT</hono.version>
```

Comment:
Shouldn't this be `1.3.0-M2` here (as in the `hono-mqtt-protocol-gateway` module)?

Reply:
Yes. Ultimately both should use the same Hono version. The problem is currently that it uses a demo certificate that is not present in `1.3.0-M2`.

Reply:
But it is now in `1.3.0-M3`, right?

Reply:
Yes.